### ISSUE TITLE
Wiring diagram

### ISSUE THREAD

**Issue by FabianRig** [FabianRig/ArduinoMoonBoardLED#20]
@FabianRig 

I'm an electrical newb, so would you mind checking this diagram since you offered above? I just called the power lines positive and negative for simplicity since it seems like they can have different labels depending on the product. Hope it's clear enough.

Another question, you mentioned putting the resistor between the Arduino and LED data -- does the wattage rating matter for the resistor, or what do you recommend / use in your system?

![image](https://github.com/FabianRig/ArduinoMoonBoardLED/assets/31644762/9e5db684-a728-445d-965d-1ed48348000a)

_Originally posted by @scott-lin in https://github.com/FabianRig/ArduinoMoonBoardLED/issues/7#issuecomment-1881972656_

**Comment by FabianRig** [comment]
@scott-lin:

I had a quick look at the wiring diagram. The wiring itself looks correct. However, I have a few remarks:
- the AC side doesn't have pos or neg but L and N, colors would be blue for N and brown for L (in Germany)
- the DC side should be +5V and gnd (ground)
- colors on the DC side should be black for gnd and red for +5V
- the LED strings usually have weird colors ;-)

You can use any kind of 330 Ohms resistor, no special resistors needed. There won't be any large currents -- otherwise the Arduino would be dead anyway.

**Comment by scott-lin** [comment]
Thanks, updated per your comments and my region's AC colors.
 

![image](https://github.com/FabianRig/ArduinoMoonBoardLED/assets/31644762/96703fc8-62c0-4c28-be3f-f09f14741ca8)

**Comment by DenkBrettl** [comment]
Thanks Scott! Would you be ok to add 230V as well? E.g. "120V or 230V power supply". 230V is the standard in a lot of the world and it might confuse some people if the diagram was very specific.

Thanks, Klaus

**Comment by scott-lin** [comment]
> Thanks Scott! Would you be ok to add 230V as well? E.g. "120V or 230V power supply". 230V is the standard in a lot of the world and it might confuse some people if the diagram was very specific.
> 
> Thanks, Klaus

Sure, I hope this update covers the gambit.

![image](https://github.com/FabianRig/ArduinoMoonBoardLED/assets/31644762/9b59765d-c397-466f-8c2a-eee34951ec52)

**Comment by DenkBrettl** [comment]
Looks great - thanks! :-)

**Comment by FabianRig** [comment]
If it's OK for you, I can include the diagram in the readme?

**Comment by scott-lin** [comment]
Yes, please feel free.

**Comment by FabianRig** [comment]
Thanks for creating and providing the wiring diagram. I've added it to the README. :-)

**Comment by tmotyl** [comment]
Thanks for the diagram. I've read somewhere that you have to connect power  every x number of leds? is that right? if yes it would be awesome to have it on the diagram too

**Comment by FabianRig** [comment]
No, it's definitely not necessary for the Moonboard LEDs. If you want to light up all LEDs, there might be a difference in color/brightness. For the handful of LEDs it's not necessary.

If you absolutely want to do double wiring, you can just connect both ends of the LED string to GND and +5V.
